### ISSUE TITLE
Request - Start to display large document on load

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#54]
```
It would be nice to load the initial pages of a large document rather than 
having to wait for the entire document to load before the viewer displays the 
contents.
```

Original issue reported on code.google.com by `faye.men...@gmail.com` on 14 Apr 2010 at 9:35

**Comment by GoogleCodeExporter** [comment]
```
progressive loading will be released in the next few weeks
```

Original comment by `erik.eng...@gmail.com` on 27 Apr 2010 at 12:46
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Expecting...
```

Original comment by `yiditu...@163.com` on 11 May 2010 at 5:08

**Comment by GoogleCodeExporter** [comment]
```
Me three!
```

Original comment by `sni...@gmail.com` on 11 May 2010 at 5:09

**Comment by GoogleCodeExporter** [comment]
```
Definitely would be a huge improvement!
```

Original comment by `BambuCha...@gmail.com` on 17 May 2010 at 6:14

**Comment by GoogleCodeExporter** [comment]
```
Implemented and available in 1.3
```

Original comment by `erik.eng...@gmail.com` on 21 Jun 2010 at 9:40
- Changed state: **Fixed**
